.tests/integration/config.yaml

```diff
@@ -3,3 +3,12 @@ units: "units.tsv"
 
 tools:
   common: "docker://marrip/common:1.1.0"
+
+reference:
+  fasta: "reference/WASH7P.fna"
+
+bwa_mem:
+  container: "docker://gmsuppsala/somatic:develop"
+  threads: 2
```

Comment:
did we decide that threads would be handled by a profile?

Reply:
I think we need to discuss because there are several ways to handle it. Unfortunately, I was mistaken and the rules do not have direct access to the profile through `snakemake`-magic. I suggest we look at the possibilities, maybe we have some other ideas as well, and then discuss what we think is the best option. I think this post could help us figure something out: https://www.sichong.site/2020/02/25/snakemake-and-slurm-how-to-manage-workflow-with-resource-constraint-on-hpc/